src/connector/src/common.rs

```diff
@@ -324,3 +325,34 @@ pub struct UpsertMessage<'a> {
     #[serde(borrow)]
     pub record: Cow<'a, [u8]>,
 }
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct NatsCommon {
```

Comment:
Looks too simple? Can you please add more options here? Such as username/password, etc, which looks very common. 

- Options for Nats client: https://docs.rs/async-nats/latest/async_nats/struct.ConnectOptions.html
- Options for JetStream: https://docs.rs/async-nats/latest/async_nats/jetstream/stream/struct.Config.html
- Maybe more.

I'm not asking for a full list of options, we may pick some basic useful according to our undertanding.

Reply:
There are many authentication methods. I only added user/password way. If more way needed, I can add it in the future. For Jetstream, I add several configurations which I thought maybe useful.